### PR TITLE
Add main alias constraint

### PR DESCRIPTION
**Pull request by itkg-nanne**
[OO-FEATURE] test on mandatory main alias in site form

### CODE REVIEW

File: Backoffice/Tests/Validator/Constraints/CheckMainAliasPresenceValidatorTest.php [hunk 1]
```diff
@@ -0,0 +1,75 @@
...
+        $this->assertInstanceOf('Symfony\Component\Validator\ConstraintValidator', $this->validator);
+    }
+
+    /**
+     * @param array $aliases
+     * @param int   $violationTimes
+     *
+     * @dataProvider provideCountAndViolation
+     */
+    public function testAddViolationOrNot($aliases, $violationTimes)
```
Comment (alavieille) [inline Backoffice/Tests/Validator/Constraints/CheckMainAliasPresenceValidatorTest.php]: array $aliases

File: Backoffice/Validator/Constraints/CheckMainAliasPresence.php [hunk 2]
```diff
@@ -0,0 +1,18 @@
+<?php
+
+namespace OpenOrchestra\Backoffice\Validator\Constraints;
+
+use Symfony\Component\Validator\Constraint;
+
+class CheckMainAliasPresence extends Constraint
```
Comment (alavieille) [inline Backoffice/Validator/Constraints/CheckMainAliasPresence.php]: php doc class

File: Backoffice/Validator/Constraints/CheckMainAliasPresenceValidator.php [hunk 3]
```diff
@@ -0,0 +1,27 @@
+<?php
+
+namespace OpenOrchestra\Backoffice\Validator\Constraints;
+
+use OpenOrchestra\ModelInterface\Model\SiteInterface;
+use Symfony\Component\Validator\Constraint;
+use Symfony\Component\Validator\ConstraintValidator;
+use OpenOrchestra\ModelInterface\Exceptions\MainAliasNotExisting;
+
+class CheckMainAliasPresenceValidator extends ConstraintValidator
```
Comment (alavieille) [inline Backoffice/Validator/Constraints/CheckMainAliasPresenceValidator.php]: phpdoc class

**Comment by alavieille** [comment]
add OO-tag

**Comment by itkg-nanne** [comment]
update done @alavieille

**Comment by alavieille** [comment]
:+1:  but can you add a context in OO-tag

**Pull request merged**